src/core/utils/qfieldcloudutils.cpp

```diff
@@ -77,6 +77,34 @@ const QString QFieldCloudUtils::getProjectId( const QString &fileName )
   return QString();
 }
 
+QString QFieldCloudUtils::userFriendlyErrorString( const QString &errorMessage )
+{
+  QString resultErrorMessage = errorMessage.startsWith( "[QF/" ) ? tr( "A server error has occured, please try again." ) : tr( "A network error has occured, please try again." );
+
+  const QString OVER_QUOTA( "over_quota" );
```

Comment:
You shouldn't have to define OVER_QUOTA twice. 

I think it'd be best to define these in qfieldcloudutils.h, i.e.:

#define ERROR_CODE_OVER_QUOTA QString( "over_quota" )

That way we insure that we ever only need to curate one list of possible code values instead of two.